src/histolab/filters/image_filters_functional.py

```diff
@@ -93,6 +93,52 @@ def rgb_to_hed(img: Image.Image) -> Image.Image:
     return hed
 
 
+def hematoxylin_channel(img: Image.Image) -> Image.Image:
+    """Obtain Hematoxylin channel from RGB image.
+
+    Input image is first converted into HED space and the the hematoxylin channel is
+    rescaled for increased contrast.
+
+    Parameters
+    ----------
+    img : Image.Image
+        Input RGB image
+
+    Returns
+    -------
+    Image.Image
```

Comment:
probably for the docstring it could be more appropriate to use `PIL.Image.Image`, regardless of how PIL is imported here
@ernestoarbitrio what do you think?

Reply:
I do agree